### ISSUE TITLE
Site Not Working

### ISSUE THREAD

**Issue by Sameer888** [xbmc-adult/xbmc-adult#196]
Please check that  porntrex,  Hq porner and Bitporno is not working

**Comment by Gujal00** [comment]
Please check if bitporno is working on the browser. Looks like it is blocked by some ISPs. It is working fine for me.
nabil did the other two sites, I think he will get to them

**Comment by Sameer888** [comment]
Yes Bitporno is working now ,thanks

**Comment by nabilbendafi** [comment]
Duplicate of #195
